tests/fafnir_tests/mod.rs

```diff
@@ -126,6 +143,10 @@ fn populate_tables(conn: &Connection) {
     conn.execute("INSERT INTO osm_aerodrome_label_point (id, osm_id, name, name_en, name_de, aerodrome_type, aerodrome, military, iata, icao, ele, geometry, tags) VALUES (30334, 1042050310, 'South Pole Station Airport',null, null, null, null, null, null,  null, null, '0101000020110F0000714501E743E172BF010000000000F87F',
      '\"name\"=>\"South Pole Station Airport\", \"aeroway\"=>\"aerodrome\", \"name_int\"=>\"South Pole Station Airport\", \"name:latin\"=>\"South Pole Station Airport\"')", &[]).unwrap();
 
+    // some lost hamlet
+    conn.execute("INSERT INTO osm_city_point (id, osm_id, name, name_en, name_de, place, population, capital, geometry, tags) VALUES (30336, 1042050311, 'I am a lost sheep',null, null, null, 3, 'somewhere', '0101000020E610000000000000000014400000000000001440',
+     '\"name\"=>\"I am a lost sheep\",\"population\"=>\"3\",\"capital\"=>\"somewhere\"')", &[]).unwrap();
```

Comment:
This test data needs to be updated to set `place=hamlet`.  
Or better, add 2 different `place` values in the table to check the SQL query works as expected.